service-request-infno-spec-1.0.0-swagger.json

```diff
@@ -0,0 +1,213 @@
+{
+  "openapi" : "3.0.0",
+  "servers" : [ {
+    "url" : "https://qaeservices1.capetown.gov.za/coct/api"
+  } ],
+  "info" : {
+    "description" : "This is a example API for Service Request.",
+    "version" : "1.0.0",
+    "title" : "Service Request Info API",
+    "license" : {
+      "name" : "Apache 2.0",
+      "url" : "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "tags" : [ {
+    "name" : "3rd_parties"
+  }, {
+    "name" : "media"
+  } ],
+  "paths" : {
+    "/service-request/status/" : {
+      "get" : {
+        "tags" : [ "3rd_parties" ],
+        "summary" : "Get service request API Info",
+        "parameters" : [ {
+          "in" : "query",
+          "name" : "service_request_no",
+          "description" : "maximum number of records to return",
+          "required" : true,
+          "schema" : {
+            "type" : "integer",
```

Comment:
rather make this a string, please.

Please also add validation of it being a 10 digit number, starting with `9`

Reply:
input type changed